overlay.d/15fcos/usr/lib/dracut/modules.d/50ignition-conf-fcos/module-setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+
+depends() {
+    echo ignition
+}
+
+install() {
+    mkdir -p "$initdir/usr/lib/ignition/base.d"
+    inst "$moddir/30afterburn-sshkeys-core.ign" \
+        "/usr/lib/ignition/base.d/30afterburn-sshkeys-core.ign"
```

Comment:
Hmm, this needs to be

```suggestion
    inst "$moddir/30-afterburn-sshkeys-core.ign" \
        "/usr/lib/ignition/base.d/30-afterburn-sshkeys-core.ign"
```

otherwise dracut won't find it, no?